schema.sql

```diff
@@ -257,7 +260,7 @@ CREATE TABLE Meeting
     module_id               int,
     subject_id              int,
     url                     varchar(200),
-    date                    date        NOT NULL,
+    date                    datetime        NOT NULL,
```

Comment:
I'm not sure that internships should have `dateTime`, @Corvette653?

Reply:
If they should, change the name of field

Reply:
I think internships should have only a date, they last two weeks so it makes no point to hold hour

Reply:
It is Meeting table, not internships.